packages/embeds/embed-core/src/embed.ts

```diff
@@ -666,6 +666,19 @@ document.addEventListener("click", (e) => {
   if (!(targetEl instanceof HTMLElement)) {
     return;
   }
+
+  /**
+   * on click, checks if the target element is the floating button
+   * if so, hides the button to ensure that it does not cover
+   * any of the modal's content.
+   *
+   * bringing the element back - check out the close() method
+   * of ModalBox element
+   */
+  if (targetEl.dataset.calNamespace === "floatingButton") {
```

Comment:
`calNamespace` can be anything. It is just a name given when there are multiple embeds on the page. Here it's floatingButton because it's been given that name through init call. So, we shouldn't check for namespace.

![Screenshot 2023-07-06 at 9.50.43 AM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/QOdRMCCW7wQwTOiuNnjo/75866528-7a45-4ddc-ad00-8ac2b555601f/Screenshot%202023-07-06%20at%209.50.43%20AM.png)



Also, I don't think we need to check if the modal is opened through floating button popup or element click. When the modal is open we should hide the floating button element so that it would never come in the way of booking.